### PR TITLE
Dash/api docs cleanup

### PR DESCRIPTION
**Pull request by pawellysy**
Fixed API docs links.

### CODE REVIEW

**Review by sebastianbochan — APPROVED** [review]

File: ts/Dashboards/Board.ts [hunk 1]
```diff
@@ -662,15 +662,15 @@ namespace Board {
          *
          * Try it:
          *
-         * {@link https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/dashboards/demo/component-highcharts/  | Highcharts component}
+         * {@link https://www.highcharts.com/samples/embed/dashboards/components/component-highcharts | Highcharts component}
```
Comment (goransle) [inline ts/Dashboards/Board.ts:665]: Sorry for being the samples nitpicker today, but here they should probably be without `embed` or `nonav`

Comment (pawellysy) [reply ts/Dashboards/Board.ts:665]: So what the URL should be? 😅

Comment (karolkolodziej) [inline ts/Dashboards/Board.ts:665]: What with the other `.ts` files in the `Dashboards` direcotry? Shouldn't we also change them?

Comment (pawelfus) [reply ts/Dashboards/Board.ts:665]: I think we should keep jsFiddle link 😅 Just like 99.99% (there is one exception) Highcharts API links redirect to jsFiddle, not to our demo pages. Unless there's a plan to change that? 🤔

File: ts/Dashboards/Board.ts [hunk 2]
```diff
@@ -662,15 +662,15 @@ namespace Board {
...
          *
-         * {@link https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/dashboards/components/component-html/ | HTML component}
+         * {@link https://www.highcharts.com/samples/embed/dashboards/components/component-html | HTML component}
          *
-         * {@link https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/dashboards/demo/component-kpi/ | KPI component}
+         * {@link https://www.highcharts.com/samples/embed/dashboards/components/component-kpi | KPI component}
          *
-         * {@link https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/dashboards/components/custom-component/ | Custom component}
+         * {@link https://www.highcharts.com/samples/embed/dashboards/components/custom-component | Custom component}
```
Comment (bre1470) [inline ts/Dashboards/Board.ts:671]: This demo looks not right.

**Review by bre1470 — APPROVED** [review]

File: ts/Dashboards/Board.ts (hunk 1 above)
Comment (goransle) [reply ts/Dashboards/Board.ts:665]: If we want to use highcharts.com/samples it should be https://www.highcharts.com/samples/dashboards/components/component-highcharts. (they're pretty similar, but `/embed` has some extra magic for being placed in iframes)

Comment (pawellysy) [reply ts/Dashboards/Board.ts:665]: Ok, let's just go with jsfiddle then 😇

**Pull request merged**